### PR TITLE
Secondary Dropdown

### PR DESCRIPTION
**Pull request by Meghana-12**
related to issue #560
Component can be viewed at `/interface/secondaryDropdown`
To test this PR, start searching with `H` in the input to view the dropdown. 
As of now, these are the entries:
Harpar Hopman
Hara Hopman
Harbour Center
Harvard University
Henry Harbour
Hara Jackson

Screenshot 2020-04-23 at 8 51 03 PM

### CODE REVIEW

File: src/ignitus-Shared/ignitus-DesignSystem/ignitus-Atoms/ignitus-secondaryDropDown/Components/secondaryDropDown.jsx [hunk 1]
```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import * as S from '../styles';
+import * as H from '../../typography';
+import { Details } from '../constants';
+
+
+export default function SecondaryDropDown() {
```
Comment (Marvin9) [inline src/ignitus-Shared/ignitus-DesignSystem/ignitus-Atoms/ignitus-secondaryDropDown/Components/secondaryDropDown.jsx:7]: To be consistent to code structure
```suggestion
export const SecondaryDropDown: React.FC = () => {
```

Change accordingly in ```Components/index.ts```

Also make an entry in ```ignitus-Shared/index.ts```,  for easy access.

Comment (Meghana-12) [reply src/ignitus-Shared/ignitus-DesignSystem/ignitus-Atoms/ignitus-secondaryDropDown/Components/secondaryDropDown.jsx]: Sure!

File: src/ignitus-Routes/ignitus-UserInterfaceBookRoutes/index.tsx [hunk 2]
```diff
@@ -93,6 +95,10 @@ export const UserInterfaceBookRoutes: React.FunctionComponent = () => (
           path="/interface/defaultDropdown"
           component={interfaceDropDown}
         />
+        <Route
+          path="/interface/secondaryDropdown"
```
Comment (Paarmita) [inline src/ignitus-Routes/ignitus-UserInterfaceBookRoutes/index.tsx:99]: I cannot see your dropdown on this route, so can you please add it. Also, try to add the information on how can we view your new PR, like mentioning the route in this case.

File: src/ignitus-Shared/ignitus-DesignSystem/ignitus-Atoms/ignitus-secondaryDropDown/Components/secondaryDropDown.tsx [hunk 3]
```diff
@@ -38,13 +36,13 @@ export default function SecondaryDropDown() {
         {showOptions &&
         <S.OptionsContainer>
           {searchResults.map((person, index) =>
```
Comment (divyanshu-rawat) [inline src/ignitus-Shared/ignitus-DesignSystem/ignitus-Atoms/ignitus-secondaryDropDown/Components/secondaryDropDown.tsx:38]: use de -structure operator to extract { name, avatarLink } in map.

```
          {searchResults.map(({ name, avatarLink }, index) => (
            <S.CardWrapper key={`${name}`}>
              <S.Avatar src={avatar} />
              <S.NameWrapper>
                <H.Heading5>{name}</H.Heading5>
              </S.NameWrapper>
            </S.CardWrapper>
          ))}
```

**Comment by Meghana-12** [comment]
> Left some feedback, though I resolved it myself but still left will be helpful foe you to enhance your skills. Overall great work 🎯 I told you , you can do ti! 🏆

Sure, I'll go through the reviews. Thank you so much

**Pull request merged**